packages/lit-html/src/lit-html.ts

```diff
@@ -233,9 +237,10 @@ export const render = (
 ) => {
   let part: NodePart = (container as any).$lit$;
```

Comment:
When given a `refNode` we should store the part on it instead of the container. That way we could render multiple times to the same container. I think it'd be confusing if a second call to render with a different `refNode` only updated the previous call's DOM.

Reply:
I think this should be addresses. This will give unexpected results, IMO:

```ts
render(templ1(), container, {refNode: a});
render(templ2(), container, {refNode: b});
```

Reply:
Done, test added.